tests/tests.py

```diff
@@ -350,6 +350,40 @@ def _create_issue_search_results_json(issues, **kwargs):
     }
 
 
+DEFAULT_NEW_REMOTE_LINK_OBJECT = {"url": "http://google.com", "title": "googlicious!"}
+
+
+class ClientRemoteLinkTests(JiraTestCase):
+    def setUp(self):
+        JiraTestCase.setUp(self)
+        self.issue_1 = self.test_manager.project_b_issue1
```

Comment:
```suggestion
        self.issue_1_key = self.test_manager.project_b_issue1
```
I also saw but this indicates this being a string and not an object of type Issue